### PR TITLE
[17.0][MIG] sale_timesheet_line_exclude

### PR DESCRIPTION
**Pull request by MohamedOsman7**

### CODE REVIEW

File: pandoc-3.1.12.1-1-amd64.deb
Comment (CRogos) [inline pandoc-3.1.12.1-1-amd64.deb:1]: Remove this file

**Review by CRogos — CHANGES_REQUESTED** [review]
Functional and Code Review LGTM, but the .deb file needs to be removed.

**Review by CRogos — APPROVED** [review]

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Comment by CRogos** [comment]
@alexey-pelykh can you also merge this?

**Pull request merged**